content/platforms/kubernetes/kubernetes-architecture.md

```diff
@@ -1,44 +1,55 @@
 
-On the left-hand side (marked #1), Redis Enterprise uses local ephemeral storage for durability. When a Pod fails, Kubernetes launches another Pod as a replacement, but this Pod comes up with empty local ephemeral storage, and the data from the original Pod is now lost.
+On the left-hand side (marked #1), Redis Enterprise uses local ephemeral storage for durability. When a Pod fails, the Kubernetes scheduler launches another Pod as a replacement. When this Pod comes up, it will likely be on another K8s node with empty local ephemeral storage and the data from the original Pod is now lost.
 
-On the right-hand side of the figure (marked #2), Redis Enterprise uses network-attached storage for data durability. In this case, when a Pod fails, Kubernetes launches another Pod and automatically connects it to the storage device used by the failed Pod. Redis Enterprise then instructs the Redis Enterprise database instance/s running on the newly created node to load the data from the network-attached storage, which guarantees a durable setup.
+On the right-hand side of the figure (marked #2), Redis Enterprise uses network-attached storage for data durability. In this case, when a Pod fails, the Kubernetes scheduler launches another Pod and automatically connects the same storage device used by the failed Pod. Redis Enterprise then instructs all the database instances running on the newly created node to load the data from the network-attached storage. This process guarantees a durable setup.
 
-Redis Enterprise is not only great as an in-memory database but also extremely efficient in the way it uses persistent storage, even when the user chooses to configure Redis Enterprise to write every change to the disk. Compared to a disk-based database that requires multiple interactions (in most cases) with a storage device for every read or write operation, Redis Enterprise uses a single IOPS, in most cases, for a write operation and zero IOPS for a read operation. As a result, significant performance improvements are seen in typical Kubernetes environments, as illustrated in the figures below:
+Redis Enterprise is not only great as an in-memory database but also extremely efficient in the way it uses persistent storage, even when the user chooses to configure Redis Enterprise to write every change to the disk. Compared to a disk-based database that requires multiple interactions (in most cases) with a storage device for every read or write operation, Redis Enterprise uses a single IOPS, in most cases, for a write operation and zero IOPS for a read operation. As a result, the performance of Redis Enterprise is not impacted significantly by using network-based storage. Moreover, significant performance improvements over other similarly deployment databases are typical in Kubernetes environments, as illustrated in the figures below:
```

Comment:
@alexmilowski `the performance of Redis Enterprise is not impacted significantly by using network-based storage`
We require block persistent storage be attached. This implies we can provide the same performance with a SAN or NAS, which isn't necessarily the case.

Reply:
"As a result, the performance of Redis Enterprise is not impacted in the same way more I/O intensive databases are by using network-based storage."

That is the point, right?

Reply:
"by using Kubernetes persistent volumes" (and implicitly, those should be block storage devices).